src/frontend/src/eth/services/eth-listener.services.ts

```diff
@@ -22,11 +19,11 @@ export const initTransactionsListener = ({
 	address: EthAddress;
 }): WebSocketListener => {
 	if (isSupportedEthTokenId(token.id)) {
-		return initEthPendingTransactionsListenerProvider({
+		return initMinedTransactionsListenerProvider({
```

Comment:
By doing so, aren't we removing the display of pending transaction that are not yet mined?